rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -291,53 +283,90 @@ describe("ActivityList", () => {
       );
       await confirmDeleteButton.trigger("click");
       await wrapper.vm.$nextTick();
+
       const modal = wrapper.findComponent({ ref: "bulkexecdeleteresult" });
       expect(modal.exists()).toBe(true);
+
       const modalConfirmDeleteButton = modal.find(
         '[data-testid="confirm-delete"]',
       );
+
       expect(modalConfirmDeleteButton.exists()).toBe(true);
 
       await modalConfirmDeleteButton.trigger("click");
+      const executionBulkDeleteSpy = jest
+        .spyOn(
+          wrapper.vm.rundeckContext.rundeckClient as any,
+          "executionBulkDelete",
```

Comment:
@smartinellibenedetti , Can you please check if it is the right way I have mocked the executionBulkDelete. It is giving error `Error: Property `executionBulkDelete` does not exist in the provided object`

Reply:
The error message is literal - the mock for rundeckContext returns an object for rundeckClient that does not contain executionBulkDelete.  It's necessary to adjust the mock.